code/_core/datum/damagetype/ranged/bullet/sniper_50.dm

```diff
@@ -13,7 +13,7 @@
 		BLUNT = 90
 	)
 
-	falloff = VIEW_RANGE + ZOOM_RANGE*5
+	falloff = VIEW_RANGE + ZOOM_RANGE*2
```

Comment:
why was the falloff nerfed here?

Reply:
wanted to differentiate between las sniper and heavy sniper, las sniper has the old faloff value, change them to be equal?